.CI/build_all

```diff
@@ -10,6 +10,7 @@ if [ -z "$GH_TOKEN" ]; then
     python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
     conda config --set show_channel_urls true
     conda config --add channels conda-forge
+    conda config --add channels r
```

Comment:
👍 for packaging R. This will make it easier to upgrade. Other communities like bioconda are eager to have easy access to it. So, packaging it here sounds like a great way to do that.